test-data/unit/fine-grained.test

```diff
@@ -8670,6 +8670,36 @@ def f(x: int) -> None: ...
 from typing import Union
 def f(x: Union[int, str]) -> None: ...
 
+[targets c, b]
+[targets2 a]
```

Comment:
The numbering doesn't seem to be in sync. I think that it would be clearer if `[targets2 ...]` would correspond to change `c.py.2` and `[targets3 ...]` would correspond to `a.py.3` (i.e. increase numbers by one).

Reply:
OK